.eslintrc.js

```diff
@@ -105,6 +105,13 @@ module.exports = {
     ],
     '@typescript-eslint/no-use-before-define': 'error',
     '@typescript-eslint/prefer-ts-expect-error': 'error',
+    '@typescript-eslint/strict-boolean-expressions': [
+      'error',
+      {
+        allowNullableBoolean: true,
+        allowNullableString: true,
```

Comment:
This can be discussed. I figured to keep allowing those checks for now as quite many changes would be necessary.
``allowNullableBoolean` allows us to do the following:
``` ts
const myBoolean: boolean | null = null
if (myBoolean) {
  // do stuff
}
```
while ``allowNullableString` allows us to do the following:
``` ts
const myString: string | null = null
if (myString) {
  // do stuff
}
```
Both are potentially unsafe since it treats `null` and `false`/`null` and empty strings `''` the same way. I don't really see a problem for `allowNullableBoolean` but `allowNullableString` might lead to a bug at some point. I still think the risk is acceptable. 

Disallowing `allowNullableBoolean` leads to 89 errors while `allowNullableString` leads to 247 errors.